ansible/playbooks/windows.yml

```diff
@@ -427,9 +428,40 @@
         remote_src: True
       tags: ant
 
+    ###################################
+    # Visual Studio 2010 Professional #
+    ###################################
+    - name: Test if VS 2010 is already installed
+      win_stat:
+        path: 'C:\Program Files (x86)\Microsoft Visual Studio 10.0'
+      register: vs2010_installed
+      tags: visual_studio_2010
+
+    - name: Download Visual Studio 2010 Professional
+      win_get_url:
+        url: 'http://download.microsoft.com/download/D/B/C/DBC11267-9597-46FF-8377-E194A73970D6/vs_proweb.exe'
+        dest: 'C:\temp\vs2010_professional.exe'
+      when: (vs2010_installed.stat.exists == false)
+      tags: visual_studio_2010
+
+    - name: Install Visual Studio 2010 Professional
+      raw: 'C:\temp\vs2010_professional.exe /q /norestart'
+      when: (vs2010_installed.stat.exists == false)
+      tags: visual_studio_2010
+
+    - name: Reboot machine after Visual Studio 2010 installation
+      win_reboot:
+      when: (vs2010_installed.stat.exists == false)
+      tags: visual_studio_2010
+
```

Comment:
In case of an issue, we don't want the playbook to hang forever. so there should be a timeout on reboot.
example:      
``` 
- name: Reboot machine after Visual Studio 2010 installation
  win_reboot:
    shutdown_timeout: 3600
    reboot_timeout: 3600
  when: (vs2010_installed.stat.exists == false)
  tags: visual_studio_2010
```

Reply:
Added a 30 mins timeout, should be enough. The maximum clock time for reboot is actually twice this value.